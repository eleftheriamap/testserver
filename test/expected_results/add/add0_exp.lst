
test/exp/test/gen/add/add0_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	517454c6 	sub	w6, w6, #0xd15, lsl #12
   4:	d15b916b 	sub	x11, x11, #0x6e4, lsl #12
   8:	0b0b00c8 	add	w8, w6, w11
   c:	8a000000 	and	x0, x0, x0
