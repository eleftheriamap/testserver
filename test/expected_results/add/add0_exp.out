Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 00000000ff2eb000
X07    = 0000000000000000
X08    = 00000000fec07000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = ffffffffff91c000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 000000000000000c
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x517454c6
0x00000004 : 0xd15b916b
0x00000008 : 0x0b0b00c8
0x0000000c : 0x8a000000
